### DISCUSSION TITLE
HTTP 500 Internal Server Error in Keyclock Integration testing on intel

### DISCUSSION THREAD

**Discussion by Chandranana** [keycloak/keycloak#25255] [Q&A]
Hi Team,

I am trying to build Keycloack on x86 environment and it is resulting in below error:
-- [ERROR]   Run 1: RealmTest>AbstractKeycloakTest.beforeAbstractKeycloakTest:181->beforeAbstractKeycloakTestRealmImport:121->AbstractKeycloakTest.removeAllRealmsDespiteMaster:473 » Processing jakarta.ws.rs.InternalServerErrorException: HTTP 500 Internal Server Error

Could you please help me resolve this error?

**Comment by Chandranana** [comment]
Any update on this?

**Comment by kaustabhbarman** [comment]
I am having same issue while building it on my M2 Mac. Any idea on how to resolve it?
